client/app/components/packages/landuse-form/public-facilities.hbs

```diff
@@ -0,0 +1,340 @@
+{{#let @form as |form|}}
+  {{#let (intersect
+    (map-by "dcpActioncode" form.data.data.landuseActions)
+    (array 'PC' 'PQ' 'PS' 'PX')
```

Comment:
Oh nifty -- So does this only display this content if `form.data.data.landuseActions` includes at least one of the four options in this array of action codes?

Reply:
Yep, exactly! I think if this was any longer or messier I would definitely move this into the component class (js) file..

Reply:
Yeah I have a computed property that does the same thing but the list of required actions codes is much longer. This is a cool way to do it though! Didn't know it was possible

Reply:
It took me a whiiile to think of this 🙈  I tried to overcomplicate at first with a new helper/component